amazon-cognito-identity-js incompatible version

## This is:
* Bug

## Specifications
* version: 5.11.0
* OS: *
* Browser: *

## Expected Behavior
The amazon-cognito-identity-js versions of the root package.json and the one required by amplify should be compatible

## Actual Behavior
I'm using @aws-amplify/auth (^4.0.2)  in one of my packages, after upgrading to 5.11.0 the packages doesn't build anymore

` ../../node_modules/@webiny/app-security-cognito-authentication/node_modules/amazon-cognito-identity-js/index.d.ts(1,1): error TS6200: Definitions of the following identifiers conflict with those in another file: ClientMetadat
a, AuthenticationDetails, CognitoUser, CognitoUserAttribute, CognitoUserPool, CognitoUserSession, CognitoAccessToken, CognitoIdToken, CognitoRefreshToken, CookieStorage, UserAgent, appendToCognitoUserAgent, WordArray`

## Detailed Description
@aws-amplify/auth (^4.0.2 -> 4.1.3) has a depenedeny to "amazon-cognito-identity-js": "^5.0.0", but the root package.json has a dependeny to "amazon-cognito-identity-js": "^4.5.3". Those two versions seem not to be compatible.

--- a/packages/app-admin-cognito/package.json
+++ b/packages/app-admin-cognito/package.json
@@ -13,7 +13,7 @@
   "license": "MIT",
   "dependencies": {
     "@apollo/react-hooks": "^3.1.5",
-    "@aws-amplify/auth": "^4.3.10",
+    "@aws-amplify/auth": "^5.1.9",
     "@emotion/styled": "^10.0.27",
     "@webiny/app": "0.0.0",
     "@webiny/app-admin": "0.0.0",

--- a/packages/app-cognito-authenticator/package.json
+++ b/packages/app-cognito-authenticator/package.json
@@ -12,7 +12,7 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "@aws-amplify/auth": "^4.0.2",
+    "@aws-amplify/auth": "^5.1.9",
     "lodash.get": "^4.4.2",
     "react": "17.0.2",
     "react-dom": "17.0.2"

--- a/packages/app-cognito-authenticator/src/hooks/useForgotPassword.ts
+++ b/packages/app-cognito-authenticator/src/hooks/useForgotPassword.ts
@@ -1,5 +1,5 @@
 import { useCallback, useReducer } from "react";
-import Auth from "@aws-amplify/auth";
+import { Auth } from "@aws-amplify/auth";
 import { useAuthenticator } from "./useAuthenticator";
 
 interface SetPasswordParams {

--- a/packages/app-cognito-authenticator/src/hooks/useRequireNewPassword.ts
+++ b/packages/app-cognito-authenticator/src/hooks/useRequireNewPassword.ts
@@ -1,5 +1,5 @@
 import { useCallback } from "react";
-import Auth from "@aws-amplify/auth";
+import { Auth } from "@aws-amplify/auth";
 import get from "lodash.get";
 import { useAuthenticator } from "./useAuthenticator";
 

--- a/packages/app-cognito-authenticator/src/hooks/useSetNewPassword.ts
+++ b/packages/app-cognito-authenticator/src/hooks/useSetNewPassword.ts
@@ -1,5 +1,5 @@
 import { useCallback, useReducer } from "react";
-import Auth from "@aws-amplify/auth";
+import { Auth } from "@aws-amplify/auth";
 import { useAuthenticator } from "./useAuthenticator";
 
 export interface UseSetNewPasswordCallableParams {

--- a/packages/app-cognito-authenticator/src/hooks/useSignIn.ts
+++ b/packages/app-cognito-authenticator/src/hooks/useSignIn.ts
@@ -1,5 +1,5 @@
 import { useCallback, useReducer } from "react";
-import Auth from "@aws-amplify/auth";
+import { Auth } from "@aws-amplify/auth";
 import { useAuthenticator } from "./useAuthenticator";
 
 export interface UseSignInCallableParams {

--- a/packages/app-security-access-management/README.md
+++ b/packages/app-security-access-management/README.md
@@ -51,7 +51,7 @@ A simple `Authenticator` React component (uses Amazon Cognito and AWS Amplify's 
 
 ```tsx
 import React, { useEffect } from "react";
-import Auth from "@aws-amplify/auth";
+import { Auth } from "@aws-amplify/auth";
 import { useSecurity, SecurityIdentity } from "@webiny/app-security";
 
 // Apart from the React component, we also configure the Auth class here.

--- a/packages/app-security/README.md
+++ b/packages/app-security/README.md
@@ -51,7 +51,7 @@ A simple `Authenticator` React component (uses Amazon Cognito and AWS Amplify's 
 
 ```tsx
 import React, { useEffect } from "react";
-import Auth from "@aws-amplify/auth";
+import { Auth } from "@aws-amplify/auth";
 import { useSecurity, SecurityIdentity } from "@webiny/app-security";
 
 // Apart from the React component, we also configure the Auth class here.

